### PR TITLE
[UsePythonVersionV0 UseRubyVersionV0] Add telemetry tracking

### PR DESCRIPTION
**Pull request by vsafonkin**
**Task name**: UsePythonVersionV0, UseRubyVersionV0

**Description**: added telemetry tracking

**Documentation changes required:** (Y/N) no

**Added unit tests:** (Y/N) no

internal issue [#1406](https://github.com/actions/virtual-environments-internal/issues/1406)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by maxim-lobanov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

File: Tasks/UseNodeV1/task.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 174,
-        "Patch": 1
+        "Minor": 178,
```
Comment (anatolybolshakov) [inline Tasks/UseNodeV1/task.json:19]: Could you please bump task version?

Comment (vsafonkin) [reply Tasks/UseNodeV1/task.json]: Done

**Review by egor-bryzgalov — CHANGES_REQUESTED** [review]

**Comment by egor-bryzgalov** [comment]
@maxim-lobanov Could you please create a separate PR for the UseNodeV1 task so we can merge other changes?

File: Tasks/UseRubyVersionV0/package.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "@types/node": "^6.0.101",
     "@types/q": "^1.0.7",
     "azure-pipelines-task-lib": "^2.9.3",
+    "azure-pipelines-tasks-utility-common": "^2.0.0",
     "vsts-task-tool-lib": "^0.5.1"
```
Comment (egor-bryzgalov) [inline Tasks/UseRubyVersionV0/package.json:30]: Could you please change it to `"azure-pipelines-tool-lib": "^0.12.0"` and update `package-lock.json` and import statements also?

**Review by egor-bryzgalov — CHANGES_REQUESTED** [review]

File: Tasks/UseRubyVersionV0/package.json (hunk 2 above)
Comment (maxim-lobanov) [reply Tasks/UseRubyVersionV0/package.json]: @egor-bryzgalov done

**Review by egor-bryzgalov — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**